File: java/gpt/p133.java
import org.springframework.boot.SpringApplication;              // 1) import app runner
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.bind.annotation.GetMapping;       // 1) import web annotations
import org.springframework.web.bind.annotation.RestController;

@SpringBootApplication
public class DemoApplication {

    public static void main(String[] args) {                    // 6) run the application
        SpringApplication.run(DemoApplication.class, args);
    }

    @RestController
    static class HomeController {

        @GetMapping("/")                                        // 3) URL -> function
        public String home() {                                   // 4) function for that URL
            return "Hello, World!";                              // 5) return message
        }
    }
}

File: python/gpt/p133.py
from flask import Flask  # 1) import web application class

app = Flask(__name__)    # 2) create application object

@app.route("/")          # 3) decorator: URL -> function
def home():              # 4) function for that URL
    return "Hello, World!"  # 5) return message

if __name__ == "__main__":
    app.run(debug=True)  # 6) run the application
